Modelica/Electrical/Analog/Lines/M_OLine.mo

```diff
@@ -348,7 +347,7 @@ the auxiliary segment_last. For the example with 4 lines the schematic of
 </blockquote>
 
 <p>The number of the capacitors and conductors depends on the number of single lines that are used,
-because each line is connected to every other line by both a capacitor and a conductor. 
+because each line is coupled to every other line by both a capacitor and a conductor. 
```

Comment:
If it a Modelica connect statement, I'd leave it as `connected`, which also is used in other descriptions.